### PR TITLE
[NO GBP] Minor tesh food changes + kiri alcohol

### PR DESCRIPTION
**Pull request by MidoriWroth**
## About The Pull Request

This PR will:
- Make muli pods produce more juice, as the amount they could produce was often not enough for food demands. You also get the same amount of juice from them whether they are ground or juiced, since they are essentially a hollow juice container.
- Piru dough and flat piru dough can be baked in both an oven and on a griddle
- Updated some minor food flavor text
- Add shakiri, a sweet drink produced from fermenting kiri fruits, along with 3 cocktails to go along with them. I meant to have this in the original pr but forgor to include it

image

## How This Contributes To The Skyrat Roleplay Experience

Gives more depth to my teshari food and allows people to have more fun with them in making drinks as well.

## Changelog

:cl:
add: Added shakiri, a sweet alcohol produced from fermenting kiri fruits, along with 3 cocktails using it.
balance: Muli pods produce more juice, and give the same amount of juice whether ground or juiced
fix: Updated some minor teshari food flavor text
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -515,3 +515,52 @@
...
+	name = "Shakiri Rogers"
+	description = "A take on the classic Roy Rogers, with shakiri instead of grenadine. Sweet and refreshing."
+	color = "#6F2B1A"
+	quality = DRINK_GOOD
+	boozepwr = 45
+	glass_icon = 'modular_skyrat/master_files/icons/obj/drinks.dmi'
+	glass_icon_state = "shakiri_rogers"
+	glass_name = "glass of shakiri rogers"
+	glass_desc = "A take on the classic Roy Rogers, with shakiri instead of grenadine. Sweet and refreshing."
+	taste_description = "fruity, carbonated soda with a slight kick"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/alcohol_reagents.dm:566]: We'll need pictures of those new drinks in your PR body.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Your PR also seems to fail some integration tests, related to the Shakiri failing to be mixed.

**Comment by GoldenAlpharex** [comment]
Also, in the future, please don't add features in a PR that's marked for NO GBP due to you fixing bugs you caused.

File: modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 2]
```diff
@@ -515,3 +515,52 @@
 	glass_desc = "Looking at it makes you recall every mistake you’ve made."
 	quality = DRINK_VERYGOOD //takes extra effort
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/consumable/ethanol/shakiri
+	name = "Shakiri"
+	description = "A sweet, fragrant red drink made from fermented kiri fruits. It seems to gently sparkle when exposed to light."
+	boozepwr = 45
+	color = "##cf3c3c"
```
Comment (Tastyfish) [inline modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/alcohol_reagents.dm]:
```suggestion
	color = "#cf3c3c"
```
This is why shakiri can't mix successfully.

**Review by Tastyfish — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 2 above)
Comment (MidoriWroth) [reply modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: oops! my bad! thanks for catching that, couldn't figure out why it was failing haha

**Review by Tastyfish — APPROVED** [review]

**Review by GoldenAlpharex — APPROVED** [review]

**Pull request merged**